### ISSUE TITLE
Multiple missing Algo's

### ISSUE THREAD

**Issue by Rimplestilhskin** [Sniffdog/Sniffdogminer#205]
Good day, is there a way I can add the missing logarithms to several pools or have 1 central file that can be updated / queried and then tested against a pool.

Using several alternative software that is similar too / clones of Sniffdogminer that can mine M7M and has more updated pools, latest mining software ... but still prefer to use SniffDogMiner vs the other multiminer / multipool mining software.

So how can we help or is there a script update example page / dictionary / (to add this algo / mining software / ??? / follow steps a-z in these files to correctly add your changes)

Similar with missing pools in EBWF on several multi mining software.

Example: 
M7M is on Ahashpool, etc ..... but not in "pools scripts" in latest release 4.8.5

Excuse a few error in this list below:
Bastion,bitcore,blakecoin,blake,blake2b,blake2s,bmw,c11,CryptoLight,CryptoNight,cryptonightv7,darkcoin-mod,decred,deep,Dmd-gr,equihash,ethash,Ethash_Blake2b,Ethash_Blake14r,Ethash_Keccak,Ethash_pascal,Ethash_Lbry,ethereum,fresh,flax,fugue,fugue256,groestl,heavy,hsr,hmq1725,jackpot,keccak,keccakc,lbry,luffa,lyra2,lyra2re,lyra2rev2,lyra2v2,lyra2z,m7m,mjollnir,Myr-gr,MyriadGroestl,neoscrypt,nist5,penta,primesr,phi,pascal,polytimos,quark,qubit,scrypt,scryptN,Scrypt-jane,SCRYPTJANE13,SCRYPTJANE14,SCRYPTJANE15,Sha-3,sha256,sha256t,sia,sib,skein,skein2,skunk,skunkhash,timetravel,tribus,vanilla,veltor,whirlpool,wildkeccak,x11,x11evo,x11gost,x12,x13,x14,x15,x16r,x16s,x17,xevan,yescrypt,yescryptR16,yescryptr32,zr5

Thanks for the help and hard work, great software.

**Comment by milltertime** [comment]
I don't think this does m7m anymore.  I had to download cpuminer-opt-3.8.7.2-windows and run it along side sniff dog as m7m (I think does not support gpu mining)
